lib/galaxy/tools/toolbox/base.py

```diff
@@ -639,6 +585,11 @@ def _load_tool_tag_set( self, elem, panel_dict, integrated_panel_dict, tool_path
         except Exception:
             log.exception( "Error reading tool from path: %s" % path )
 
+    def _get_tool_shed_repository( self, tool_shed, name, owner, installed_changeset_revision ):
+        # Abstract class does't have a dependency on the database, for full tool shed
```

Comment:
What are the consequences when you remove `__get_tool_shed_repository()` and don't implement the old DB version?